azerothcore/azerothcore-wotlk
# fix(DB/Creature) Removes NPCs that shouldn't exist After v. 2.1.0

## Changes Proposed:

This PR proposes changes to:

Removes:
- Terror-Fire Guardian // 22327
- Nightmare Imp // 22202
- Terrordar the Tormentor // 22385
- ~Bladespine Basilisk~ (already doesn't exist, 22187)
- Wrath Reaver // 22196
- Wrath Fiend // 22392
- Fel Rager // 22286
- Trigul // 22174
- Felstorm Corruptor // 22217
- Insidious Familiar // 22218
- Darkflame Infernal // 22289
- Felstrom Overseer // 22221
- Eredar Sotrmbringer // 22283
- Amberpelt Clefthoof // 22287
Total found and removed of `124` rows

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/6317
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/20820

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



```sql
SELECT * FROM creature WHERE id1 IN (22202, 22385, 22187, 22196, 22392, 22286, 22174, 22217, 22218, 22289, 22221, 22283, 22287, 22327)
```
- Fly over `Blade's Edge Mountains` area and see the missing creatures now

More specific

> Go to Blade's Edge Mountains zone and fly to Forge Camp Terror
> Fly over the zone, paying attention to the present mobs
> Fly to Forge Camp Wrath
> Fly over the zone, paying attention to the present mobs
> Fly to Crystal Spine
> Fly over the zone, paying attention to the present mobs
> Fly to Bash'ir Landing
> Fly over the zone, paying attention to the present mobs

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
